main/src/main/java/cgeo/geocaching/ui/CompassMiniView.java

```diff
@@ -88,7 +88,7 @@ public void setTargetCoords(final Geopoint point) {
         targetCoords = point;
     }
 
-    void updateAzimuth(final float azimuth) {
+    public void updateAzimuth(final float azimuth) {
```

Comment:
You might want to take this opportunity to rewrite CompoassMiniView in a self-contained way: maybe this class should by itself register and unregister from geoposition and direction updates and not rely on being called for updates by the outside world?